### PR TITLE
register gui

### LINKED ISSUE #147: UI - add feature to register a new managed service
1. Add a new option to side bar called -> Register new service.
2. Generate REST client from the swagger of xpanse project in GUI
3. Use register method to import a JSON File. 
4. with import -> display the contents of the file using the data types from the open api. 
5. Call API to register the managed service. 
6. Display success or failure message based on API response. 
7. In case of error -> display all validation errors returned by API:
image

### PR DESCRIPTION
**Pull request by swaroopar**
Fixed #147

### CODE REVIEW

**Review by Alice1319 — APPROVED** [review]
looks good.

**Review by iskey — CHANGES_REQUESTED** [review]

File: ui/src/components/content/login/LoginScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ */
+
+import { LockOutlined, UserOutlined } from '@ant-design/icons';
+import { Form, Image, Input, message, Modal, Space } from 'antd';
+import { useLocation, useNavigate } from 'react-router-dom';
+import { homePageRoute, isAuthenticatedKey, usernameKey } from '../../utils/constants';
+import { useForm } from 'antd/es/form/Form';
+
+function LoginScreen(): JSX.Element {
+    const validUsers: Array<string> = ['csp', 'user', 'otc'];
```
[the remark this answers is not in the document]
Comment (swaroopar) [reply ui/src/components/content/login/LoginScreen.tsx:13]: we need for sure. .but later. we dont have any authentication and authorization concept yet.

File: ui/src/components/content/register/DisplayOclData.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+                                />
+                            }
+                        </Descriptions.Item>
+                        <Descriptions.Item label='Flavours'>{<pre>{getFlavoursText(ocl)}</pre>}</Descriptions.Item>
```
[the remark this answers is not in the document]
Comment (swaroopar) [reply ui/src/components/content/register/DisplayOclData.tsx]: updated. mix up between UK and US english

File: ui/src/components/content/login/LoginScreen.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (iskey) [reply ui/src/components/content/login/LoginScreen.tsx:13]: sure.

**Pull request merged**